ui/apps/platform/src/Containers/Clusters/DelegateScanning/DelegateScanningPage.tsx

```diff
@@ -32,21 +42,63 @@ function DelegateScanningPage() {
     const displayedPageTitle = 'Delegate Image Scanning';
     const [delegatedRegistryConfig, setDedicatedRegistryConfig] =
         useState<DelegatedRegistryConfig>(initialDelegatedState);
-    const [errMessage, setErrMessage] = useState<string>('');
-
-    const { clusters } = useFetchClustersForPermissions(['Cluster']);
+    const [delegatedRegistryClusters, setDelegatedRegistryClusters] = useState<
+        DelegatedRegistryCluster[]
+    >([]);
+    const [alertObj, setAlertObj] = useState<AlertObj | null>(null);
 
     useEffect(() => {
-        setErrMessage('');
+        fetchClusters();
+        fetchConfig();
+    }, []);
+
+    function fetchConfig() {
+        setAlertObj(null);
         fetchDelegatedRegistryConfig()
             .then((configFetched) => {
                 setDedicatedRegistryConfig(configFetched);
             })
             .catch((error) => {
-                setErrMessage(getAxiosErrorMessage(error));
+                const newErrorObj: AlertObj = {
+                    type: AlertVariant.danger,
```

Comment:
Maybe worth discussion among UI team and then with UX team about pro and con of **warning** for GET request failures and **danger** for PUT request failures. For example, traffic signals have hierarchy:
* Red means stop at intersection, stop for train, stop for school bus, or give way for fire or medic.
* Yellow includes slow down in school zone, even though you will pay points if you get caught speeding.

Reply:
There is a timing condition which error wins if both requests fail. I guess flip of the coin is okay?

Reply:
1. Regarding a rule of thumb for warn vs. error, I agree with what you've proposed, but this case is an exception, in that if either call failed, the page is unusable. The config is created at install, and to even view, let alone edit it, you need both the config object itself AND the list of appropriate clusters. The only case where I can see this fail is either a temporary network blip (in which case, try refreshing the page), or the API has gone down since you loaded the UI SPA, so there will be a warning banner at the top of the whole app soon.
2. Regarding the timing condition, I saw that, and considered experimenting with an array for the error messages, but then considered the points I just listed in number 1 above, thought that was overkill, because they both need to succeed to use the page.

I'll note in the PR description to disable the Save button if there is any error, when I add form error handling in general (either through Formik or locally).